tendermint/tests/integration.rs

```diff
@@ -28,7 +29,12 @@ mod rpc {
     #[test]
     #[ignore]
     fn abci_query() {
-        // TODO(tarcieri): write integration test for this endpoint
+        let key = Path::from_str("unpopulated_key").unwrap();
```

Comment:
This could alternatively be:

```suggestion
        let key = "unpopulated_key".parse().unwrap();
```

Reply:
✅